libs/game---hw/gameoverrides.ts

```diff
@@ -1,18 +0,0 @@
-/**
- * Repeats the code forever in the background. On each iteration, allows other codes to run.
- * @param body code to execute
- */
-//% help=loops/forever weight=100 afterOnStart=true blockNamespace="loops"
-//% blockId=forever block="forever" blockAllowMultiple=1
-function forever(a: () => void): void {
```

Comment:
are these lost?

Reply:
there were two copies (another one in game/gameoverrides.ts); now there's one